.gitignore

```diff
@@ -13,7 +14,16 @@ logs
 /jsettlers.common/resources/images/*
 !/jsettlers.common/resources/images/movables.txt
 /jsettlers.common/resources/save
+config.prp
 
 /jsettlers.main.swing/*.log
 
-/jsettlers.tests/out
+#IDE Files
```

Comment:
I'm ok with adding the IDE files to the gitignore list. Although there is a huge number of IDEs, I think only a small number of them will be used. Maybe we should say we support Eclipse, IntelliJ and Netbeans (or what are you using @adamjryan?).